test/test__helper.rb

```diff
@@ -45,4 +45,12 @@ def load_it(judge, fb)
     $loog = Loog::NULL
     load(File.join(__dir__, "../judges/#{judge}/#{judge}.rb"))
   end
+
+  def test_date
```

Comment:
@Suban05 It seems if the method begins with `test_` - this is test case in minitest

Reply:
@Suban05 see  [this comment](https://github.com/zerocracy/judges-action/pull/277#discussion_r1715723449)